diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -168,6 +184,11 @@ class BlockStructureBraces : Rule("block-structure") {
             if (checkBraceNode(it, true)) {
                 BRACES_BLOCK_STRUCTURE_ERROR.warnAndFix(configRules, emitWarn, isFixMode, "incorrect new line after closing brace",
                         it.startOffset) {
+                    if (it.elementType != WHITE_SPACE) {
+                        it.addChild(PsiWhiteSpaceImpl(" "), it.treePrev)
```

Comment:
This line is marked as not covered by tests, please add an example for this case. Also, will this call work at all with `it.treePrev` as second argument?